### PR TITLE
Update gdk to 0.0.47

### PR DESCRIPTION
**Pull request by jgriffiths**

### CODE REVIEW

**Comment by jkauffman1** [comment]
utack 1f40b086b68f42c565b8517aeccea9e200fb7f09

File: green_cli/common.py [hunk 1]
```diff
@@ -400,7 +401,9 @@ def _txlist_summary(txlist):
             tx['satoshi'] = {asset: -tx['satoshi'][asset] for asset in tx['satoshi']}
         for asset, amount in tx['satoshi'].items():
             balance[asset] += amount
-            lines.append(f"{tx['txhash']} {tx['created_at']} ({confs}) {amount:+} "\
+            ts = tx['created_at_ts']
+            created_at = datetime.fromtimestamp(ts / 1000000.0, tz=timezone.utc).replace(tzinfo=None)
```
Comment (LeoComandini) [inline green_cli/common.py:405]: Do we want to format the date up to micro seconds? perhaps we want to do `ts // 1000000` so we round to seconds

**Comment by LeoComandini** [comment]
utACK 1f40b086b68f42c565b8517aeccea9e200fb7f09 with comment

File: green_cli/common.py (hunk 1 above)
Comment (jgriffiths) [reply green_cli/common.py]: Since this is ust for summary display, that makes sense. Updated, thanks.

**Pull request merged**